app/src/androidTest/kotlin/org/wordpress/aztec/demo/pages/EditorPage.kt

```diff
@@ -367,6 +370,25 @@ class EditorPage : BasePage() {
         return this
     }
 
+    fun hasChanges(shouldHaveChanges : AztecText.EditorHasChanges): EditorPage {
+        val hasNoChangesMatcher = object : TypeSafeMatcher<View>() {
+            override fun describeTo(description: Description) {
+                description.appendText("User has been made changes to the post: $shouldHaveChanges")
```

Comment:
super nit, it should be  `User has made changes to the post...` (remove the word "been")